include/vrapp/VolumeVisualizationApp.h

```diff
@@ -80,7 +80,7 @@ class VolumeVisualizationApp : public MinVR::VRApp
 
   std::chrono::steady_clock::time_point m_lastTime;
 
-  VRVolumeApp *m_vrVolumeApp;
+  std::unique_ptr<VRVolumeApp> m_vrVolumeApp;
```

Comment:
What's this `unique_ptr` do? Does it prevent you from having multiple pointers to the same object? I've never seen it before

Reply:
[That's something fancy from C++ 11](https://learn.microsoft.com/en-us/cpp/cpp/smart-pointers-modern-cpp?view=msvc-170) to delegate memory allocation to the compiler and free the user from that responsibility.  Im trying to make this old code a little bit more modern.